src/app/modules/admin/components/team-card/team-card.component.ts

```diff
@@ -1,15 +1,62 @@
-import { Component, OnInit } from '@angular/core';
+import {Component, OnInit, Input} from '@angular/core';
 
 @Component({
   selector: 'app-team-card',
   templateUrl: './team-card.component.html',
   styleUrls: ['./team-card.component.scss']
 })
 export class TeamCardComponent implements OnInit {
+  @Input() teamCard;
+  teamName: string;
+  members: {
+    name: string;
+    level?: string;
+    lead?: boolean;
+    profilePic: string;
+  }[];
+  levels: string;
 
   constructor() { }
 
   ngOnInit() {
+    this.teamName = this.teamCard.teamName;
+    this.members = this.teamCard.members;
+    this.countLevels();
   }
 
+  private countLevels(): void {
+    const levels = { junior: 0, middle: 0, senior: 0, unknown: 0};
+    let levelsStr = '';
+    for (const member of this.members) {
+      if (member.level === 'junior') {
+        levels.junior += 1;
+      } else if (member.level === 'middle') {
+        levels.middle += 1;
+      } else if (member.level === 'senior') {
+        levels.senior += 1;
+      } else {
+        levels.unknown += 1;
+      }
```

Comment:
Здесь можно уменьшить количество кода , используя метод filter массивов
https://learn.javascript.ru/array-iteration#filter
```suggestion
    const levels = {
    junior: this.members.filter((member)=> условие проверки и т.д.
    };
```